rxnetty-common/src/main/java/io/reactivex/netty/client/pool/PoolConfig.java

```diff
@@ -37,7 +37,7 @@
 
     public PoolConfig() {
         maxIdleTimeMillis = DEFAULT_MAX_IDLE_TIME_MILLIS;
-        idleConnCleanupTicker = Observable.timer(maxIdleTimeMillis, TimeUnit.MILLISECONDS);
+        idleConnCleanupTicker = Observable.interval(maxIdleTimeMillis, TimeUnit.MILLISECONDS);
```

Comment:
Sorry, one more. This should have an initial delay so that it doesn't try to clean up immediately. It's less important on creation, but if there are errors it will keep emitting right after each failure. This means a failure will keep triggering as fast as it can instead of just every `maxIdleTimeMillis`.

```
idleConnCleanupTicker = Observable.interval(maxIdleTimeMillis, maxIdleTimeMillis, TimeUnit.MILLISECONDS);
```